__tests__/app.test.js

```diff
@@ -53,3 +54,42 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  it("responds with a 200 status code and returns an object with the correct number of properties and with the correct structure", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Object.keys(body.article).length).toBe(8);
```

Comment:
may not be necessary to check how many keys there are - It'll be updated in the further tasks